File: pepcoding/week2/day2/arraydemo.cpp
#include<iostream>
#include<vector>
using namespace std;

void input(vector<int>& v1)
{
    int n=v1.size();
     cout<<"enter the element in vector";
        
    for(int i=0;i<n;i++)
    {
        cin>>v1[i];
       
    }
}


void output(vector<int>& v1)
{
    int n=v1.size();
     cout<<"the element in vector";
    for(int i=0;i<n;i++)
    {
       
        cout<<v1[i]<<" ";
    }
}
int main()
    {
        int n;
        cout<<"enter the size of the vector";

        cin>>n;
        vector<int> v1(n,0);
        input(v1);
        output(v1);
    }
    //g++ arraydemo.cpp -o out
    //out

File: pepcoding/week2/day2/arraydemo.java
import java.util.Scanner;





class arraydemo
{
    public static void main(String args[])
    {
        System.out.println("enter the size of array");

        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();

        int []arr=new int[n];
        input(arr);
        output(arr);

    }        
static void input(int[]arr)
{
    for(int i=0;i<arr.length;i++)
    {
        Scanner sc=new Scanner(System.in);
        int x=sc.nextInt();
        arr[i]=x;

        
    }

}

static void output(int []arr)
{
    System.out.println("the output of array");
    for(int i=0;i<arr.length;i++)
    {
        System.out.println(arr[i]);
        System.out.println();
    }
}



    }